### ISSUE TITLE
Error on generating MnemonicKey using Terra.js on react-native

### ISSUE THREAD

**Issue by xcsob** [terra-money/terra.js#212]
Hello, 
I'm trying to generate the MnemonicKey using Terra.js library on react-native, but I'm facing an error. I've been trying to do this for a while, but I cannot find any solutions. 

**Code**

**1) Configuration**

The libraries in my package.json are the following
```
 "@terra-money/terra.js": "^3.0.0",
  "node-libs-react-native": "^1.2.1",
  "react-native-get-random-values": "^1.7.1",
  "react": "^17.0.2",
  "react-native": "^0.66.0",
```

**2) My metro.config.js**
```
module.exports = {
    resolver: {
      sourceExts: ['jsx', 'js', 'ts', 'tsx'],
      extraNodeModules: require('node-libs-react-native'),
    },
};
```

**3) My index.js**
```
import 'react-native-gesture-handler';
import { registerRootComponent } from 'expo';
import 'node-libs-react-native/globals';
import 'react-native-get-random-values';

import App from './App';
registerRootComponent(App);
```

**4) Code that generates the error**
```
import { MnemonicKey } from '@terra-money/terra.js'
const mk = new MnemonicKey()  <----- Error at this line
```

**Error Received**
`Possible Unhandled Promise Rejection (id: 0): TypeError: Cannot read properties of undefined (reading 'call') TypeError: Cannot read properties of undefined (reading 'call') at Hash.CipherBase (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:284695:15) at new Hash (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:278351:10) at createHash (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:278369:12) at deriveChecksumBits (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:293366:16) at entropyToMnemonic (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:293467:24) at Object.generateMnemonic (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:293487:12) at new MnemonicKey (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:277942:26) at TerraService.prova (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:210397:9) at _callee$ (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:441072:87) at tryCatch (http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.belprogetto:24594:19)`

**Expected Behaviour**
The mk variable is an instance of object MnemonicKey.

**Note**: I already used the react-native-mnemonic-key as written in the [readme](https://terra-money.github.io/terra.js/) but on this [issue](https://github.com/terra-money/terra.js/issues/205#issuecomment-1008610423) it was answered me that it is dismissed now.

**Comment by Vritra4** [comment]
i don't know about react-native well but.. can you try https://www.npmjs.com/package/rn-nodeify?

**Comment by xcsob** [comment]
> i don't know about react-native well but.. can you try https://www.npmjs.com/package/rn-nodeify?

Thanks. 
I tried using this on an entirely new project. However, I get this other error now: 

` ERROR  TypeError: undefined is not an object (evaluating '_$$_REQUIRE(_dependencyMap[0], "stream").Transform.call')`

![Schermata 2022-01-15 alle 16 12 59 PM](https://user-images.githubusercontent.com/8886526/149626853-e7712d52-8bec-4e82-a854-f37c6c05211b.png)

The issue has been discussed also [here](https://stackoverflow.com/questions/67045804/react-native-crypto-stream-module-is-undefined), I tried the suggested solution with no luck.
